Dockerfile

```
@@ -1,23 +1,14 @@
-FROM node:dubnium-stretch-slim
+FROM alpine:latest
```

Never tag latest. Use 3.11 instead.

```suggestion
FROM 3.11
```